### ISSUE TITLE
Turn off Rendering of links in BeakerX tables

### ISSUE THREAD

**Issue by JohnOmernik** [twosigma/beakerx#8090]
Some times my data has links in it.  BeakerX apparently detects that and tries to make it a hyper link. Can we turn this off so it just shows the text without a hyperlink (so we don't accidentally click on it. )

**Comment by JohnOmernik** [comment]
Note.  it appears to be only a beakerx thing, if I to_html() or to_list and then print it in a  notebook, it doesn't auto hyperlink the URL. 

Where would I look to find a way to make this "auto interpretation of a link" an optional thing.

**Comment by JohnOmernik** [comment]
So, I found in this doc page that it is indeed doing this:

https://nbviewer.jupyter.org/github/twosigma/beakerx/blob/master/doc/python/TableAPI.ipynb

This could be very dangerous depending on the data set from a security perspective.

**Comment by gkatakou** [comment]
I agree with the above. The display of live links should at least be an option that can be enabled/disabled. I use beakerx tables to display data containing malicious URLs. I currently need to defang them in the backend pandas dataframe before displaying them in a beakerx table.

**Comment by haloforensics** [comment]
Curious if the auto link disable feature has been implemented?  If so, how would you disable auto links when using BeakerX in Python?
